prog6.rb

```diff
@@ -0,0 +1,36 @@
+# Q-6 Define a class named 'Vehicle' consisting of 'name','price' and methods for initializing 
+# and showing contents(overwrite to_s). The 'price' of Vehicle may change over time. 
+# Now create a subclass 'Bike' having 'dealer' and percent_price_increase and method to show its content. 
+# Initialize a Bike class object with certain values. Define a method price_increase which will increase the price. 
+# Your program should accept command line input in this format: BikeName Price Dealer PercentPriceIncrease. 
+# Ex: BajajDiscover 58000 BaggaLink 12
+
+class Vehicle
+  attr_accessor :name, :price
+  def initialize(name, price)
+    @name = name
+    @price = price.to_f
+  end
+end
+class Bike < Vehicle
```

Comment:
Leave one line after this. See the image:
<img width="1440" alt="Screenshot 2021-05-19 at 12 35 25 PM" src="https://user-images.githubusercontent.com/51733770/118770421-ee763580-b89e-11eb-86aa-d07469649f3c.png">